### PR TITLE
Package sedlex.3.2

### PR DESCRIPTION
**Pull request by toots**
### `sedlex.3.2`
An OCaml lexer generator for Unicode
sedlex is a lexer generator for OCaml. It is similar to ocamllex, but supports
Unicode. Unlike ocamllex, sedlex allows lexer specifications within regular
OCaml source files. Lexing specific constructs are provided via a ppx syntax
extension.

---
* Homepage: https://github.com/ocaml-community/sedlex
* Source repo: git+https://github.com/ocaml-community/sedlex.git
* Bug tracker: https://github.com/ocaml-community/sedlex/issues

---
:camel: Pull-request generated by opam-publish v2.2.0

### CODE REVIEW

File: packages/sedlex/sedlex.3.2/opam [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "An OCaml lexer generator for Unicode"
+description: """
+sedlex is a lexer generator for OCaml. It is similar to ocamllex, but supports
+Unicode. Unlike ocamllex, sedlex allows lexer specifications within regular
+OCaml source files. Lexing specific constructs are provided via a ppx syntax
+extension."""
+maintainer: ["Alain Frisch <alain.frisch@lexifi.com>"]
```
Comment (raphael-proust) [inline packages/sedlex/sedlex.3.2/opam:9]: Would it be fair to say that you are now the maintainer? Looking at the graphs from github it certainly looks that way…

**Comment by raphael-proust** [comment]
I've restarted the failed CI jobs because a large portion of them were cancelations/timeouts/networks.

I checked a few of the other ones and I didn't spot any issues related to this release, but I haven't checked them all. It'll be easier to check once the transient failures are gone.

File: packages/sedlex/sedlex.3.2/opam (hunk 1 above)
Comment (toots) [reply packages/sedlex/sedlex.3.2/opam:9]: The package and module are maintained by the `ocaml-community` group.

**Comment by toots** [comment]
> I've restarted the failed CI jobs because a large portion of them were cancelations/timeouts/networks.
> 
> I checked a few of the other ones and I didn't spot any issues related to this release, but I haven't checked them all. It'll be easier to check once the transient failures are gone.

Yes, it looks like they're all related to issues internal to them. From an API/compilation perspective, the new release should be strictly backward compatible.

File: packages/sedlex/sedlex.3.2/opam (hunk 1 above)
Comment (raphael-proust) [reply packages/sedlex/sedlex.3.2/opam:9]: Ok, I guess we can either leave the field as is (it's not very accurate but also not completely arbitrary) or update it to "ocaml-community" (is there an email address associated to that entity?). I don't mind either.

Comment (toots) [reply packages/sedlex/sedlex.3.2/opam:9]: Good question. I don't think that there's an email. I would leave it as is for now, it's something I should at least discuss with the community before making a decision.

Comment (raphael-proust) [reply packages/sedlex/sedlex.3.2/opam:9]: :+1:

**Comment by mseri** [comment]
Thanks

**Pull request merged**